ng2-components/ng2-alfresco-core/src/components/collapsable/accordion-group.component.ts

```diff
@@ -29,12 +29,18 @@ export class AccordionGroupComponent implements OnDestroy {
     private _isOpen: boolean = false;
     private _isSelected: boolean = false;
 
+    @ViewChild('contentWrapper')
+    contentWrapper: any;
+
     @Input()
     heading: string;
 
     @Input()
     headingIcon: string;
 
+    @Output()
+    headingSelect: EventEmitter<any> = new EventEmitter<any>();
```

Comment:
Hi @eromano I changed headingSelect to headingClick. And there is no documentation for Accordion Menu.